src/routes/tax.ts

```diff
@@ -65,7 +65,7 @@ router.put("/:id", async (req, res, next) => {
       return;
     }
 
-    const changed = await DAO.tax.updateTaxUnit(id, taxUnit);
+    const changed = await DAO.tax.updateTaxUnit(parseInt(id), taxUnit);
```

Comment:
Could this have side effects? I'm thinking perhaps the id could be a string in rare cases (e.g. `abc123`) which would cause an exception here.

Reply:
Mm true. I guess the question is if we want to allow string ids here or not. Do we @fellmirr ?

If so we should update `updateTaxUnit` to expect a string instead

Reply:
We do parse it as an int here though https://github.com/stiftelsen-effekt/effekt-backend/blob/master/src/routes/donors.ts#L779

Reply:
Ah ok, then this should be fine!